packages/arb-validator/cmdhelper/cmdhelper.go

```diff
@@ -139,21 +75,21 @@ func ValidateRollupChain(execName string, managerCreationFunc func(rollupAddress
 		return err
 	}
 
-	rollupActor, err := client.NewRollup(address)
+	rollupActor, err := client.NewRollup(rollupArgs.Address)
 	if err != nil {
 		return err
 	}
 
-	validatorListener := rollup.NewValidatorChainListener(context.Background(), address, rollupActor)
+	validatorListener := rollup.NewValidatorChainListener(context.Background(), rollupArgs.Address, rollupActor)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [line is 112 characters, out of limit 80](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)